convert2rhel/pkghandler.py

```diff
@@ -15,15 +15,23 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-from itertools import imap
+from builtins import str
+from builtins import range
+from builtins import object
+
 import logging
 import os
 import re
-import yum
+
+try:
+    import dnf as yum  # as yum to avoid errors accessing dnf if it's not imported
+except ImportError:
+    import yum
```

Comment:
I strongly recommend you encapsulate this in a wrapper module. This code looks kind of gnarly as it is, and the yum/dnf details make it harder to read.

Reply:
Thanks, fixed